src/docfx/build/resolve/Resolve.cs

```diff
@@ -61,7 +62,7 @@ public static (Error error, string href, string fragment, Document file) TryReso
             }
 
             // Link to dependent repo, don't build the file, leave href as is
-            if (file.Docset != relativeTo.Docset)
```

Comment:
avoid linq: != r.docset && != file.fallbackdocset?

Reply:
!= r.Docset and !=r.Fallbackdocset && !=r.LocalizationDocset and I don't want to expose this logic out of document...

---
In reply to: [225760299](https://github.com/dotnet/docfx/pull/3499#discussion_r225760299) [](ancestors = 225760299)